doc/source/rllib/rllib-dev.rst

```diff
@@ -112,7 +112,7 @@ Finding Memory Leaks In Workers
 
 Keeping the memory usage of long running workers stable can be challenging. The ``MemoryTrackingCallbacks`` class can be used to track memory usage of workers.
 
-.. autoclass:: ray.rllib.algorithms.callbacks.MemoryTrackingCallbacks
+.. autoclass:: ray.rllib.callbacks.callbacks.MemoryTrackingCallbacks
```

Comment:
Nice! This is the right place for the callbacks. Did never understand why they resided in the `algorithms` folder.

Reply:
Me, neither. That was super random :D